src/cvc.mli

```diff
@@ -0,0 +1,33 @@
+module RSA :
+sig
+  module Public :
+  sig
+    type t =
+      { n: Z.t
+      ; e: Z.t
+      }
+    [@@deriving ord,yojson,eq,show]
+  end
+end
+
+module ECDSA :
+sig
+  module Public :
+  sig
+    type t =
+      { modulus : Z.t
+      ; coefficient_a : Z.t
+      ; coefficient_b : Z.t
+      ; base_point_g : Z.t
+      ; base_point_r_order : Z.t
+      ; public_point_y : Z.t
+      ; cofactor_f : Z.t
+      }
+    [@@deriving ord,yojson,eq,show]
+  end
+end
+
+type t =
```

Comment:
Can you add a `decode` function to the `RSA` and `ECDSA` modules (that constrains the OID) and not expose this one? Thanks!